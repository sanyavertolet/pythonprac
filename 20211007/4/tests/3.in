print(Calc('sin(cos(sin(cos(x))))', 'log(4**(x * sin(x)))', 'x - y')(5))
