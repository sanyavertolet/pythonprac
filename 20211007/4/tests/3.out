7.4665730385604805
